README.md

```diff
@@ -51,38 +110,17 @@ https://github.com/learn-co-students/forks-and-clones-readme-v-000
 
```

Comment:
Above the URLS are coded like:  https://github.com/learn-co-students/forks-and-clones-readme-v-000

This is going to be brittle when shared across to other tracks. Could we have them look in their repo and navigate to the just-cloned forks-and-clones-readme? This is legacy badness, but let's fix it here as well.

Reply:
@sgharms I removed it entirely. Would it be unclear to them without those lines? Seems like there should be a more straight-foward workflow (likely using the UI) that GitHub provides than changing urls.